### PR TITLE
fix(summitbanner): display only for some subsidiaries

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Display only for some subsidiaries

MANAGER-1364

I've also included `yarn.lock` as the current version of `yarn.lock` prevents the project to work properly

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4,22 +4,19 @@
 
 "@babel/code-frame@7.0.0-beta.44":
   version "7.0.0-beta.44"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz#2a02643368de80916162be70865c97774f3adbd9"
-  integrity sha512-cuAuTTIQ9RqcFRJ/Y8PvTh+paepNcaGxwQwjIDRWPXmzzyAeCO4KqS9ikMvq0MCbRk6GlYKwfzStrcP3/jSL8g==
+  resolved "http://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0-beta.44.tgz#2a02643368de80916162be70865c97774f3adbd9"
```
Comment (FredericEspiau) [inline yarn.lock]: The current version of `yarn.lock` prevents the project to work properly

Comment (antleblanc) [reply yarn.lock]: Could you please upgrade your `yarn` version to `1.10.1`?

**Comment by ovh-cds** [comment]
CDS Report build-yarn#1727.0 ✘  
* Stage 1
  * deps ✘

File: yarn.lock (hunk 1 above)
Comment (FredericEspiau) [reply yarn.lock]: You were right (as always)

**Pull request merged**